pages/_app.tsx

```
@@ -10,6 +10,10 @@ import { NuqsAdapter } from "nuqs/adapters/next/pages";
 
 import { EXCLUDED_PATHS } from "@/lib/constants";
 
+import { EXCLUDED_PATHS } from "@/lib/constants";
```

Biome (1.9.4)

[error] Shouldn't redeclare 'EXCLUDED_PATHS'. Consider to delete it or rename it.

'EXCLUDED_PATHS' is defined here:


(lint/suspicious/noRedeclare)